packages/contracts/script/HelperConfig.s.sol

```diff
@@ -40,10 +40,8 @@ contract HelperConfig is Script {
 
         config = loadConfiguration();
 
-        if (block.chainid == 421614 || block.chainid == 80001 || block.chainid == 84532) {
-            activeNetworkConfig = getSepoliaEthConfig();
-        } else if (block.chainid == 31337) {
-            activeNetworkConfig = getAnvilEthConfig();
+        if (block.chainid == 31337 || block.chainid == 421614 || block.chainid == 80001 || block.chainid == 84532) {
```

Comment:
I think this should be in separate if else logic.
Because, we will not deploy mocks on all network. 
We deploy mocks only on anvil and testnet. (At some point for testnet also we will use already deployed address).
And for mainnet there will not be any mock deploy.

Reply:
Ok, the reason I did this is that the config functions were identical. So, I love your point, but we were not deploying to mainnet in the scripts :)  Care to pair on this?

Reply:
As discussed, this certainly needs to be done before we deloy to any mainnet. But not in this PR.